pkgdb/Makefile

```diff
@@ -244,8 +244,8 @@ ifeq (gcc,$(TOOLCHAIN))
 CXXFLAGS += -ggdb3 -pg -fno-omit-frame-pointer
 LDFLAGS  += -ggdb3 -pg -fno-omit-frame-pointer
 else # Clang
-CXXFLAGS += -g -pg
-LDFLAGS  += -g -pg
+CXXFLAGS += -g -glldb -fno-omit-frame-pointer -mno-omit-leaf-frame-pointer
```

Comment:
Do these work on gcc? I mean I guess, since the tests don't fail...

Reply:
This is explicitly the clang section

Reply:
```
else # Clang
```

lol im blind, sorry